finetune/saver.py

```diff
@@ -129,6 +126,31 @@ def before_run(self, run_context):
             self.refresh_base_model = False
 
 
+class BatchedVarLoad:
```

Comment:
We cannot add ops so assign is out of the window. Also it bloats the graph because we add all the weights once as constants and once as variables. Its different to having a bunch of calls to load because each of them invokes sess.run which is slow as hell compared to batching them up.